tests/jobs/localisation/test_config1.py

```diff
@@ -0,0 +1,831 @@
+                "PARAM_NODE22:P22",
+                "PARAM_NODE2:PARAM1",
+                "PARAM_NODE2:PARAM2",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE2Y:Y1",
+                "PARAM_NODE2Y:Y2",
+                "PARAM_NODE2Y:Y3",
+                "PARAM_NODE3:PARAM4",
+                "PARAM_SURFACE2",
+            ],
+        ),
+        (
+            ["P*2:*", "PARAM_NODE3*"],
+            [
+                "PARAM_NODE2:PARAM1",
+                "PARAM_NODE2:PARAM2",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE22:P1",
+                "PARAM_NODE22:P2",
+                "PARAM_NODE22:P22",
+                "PARAM_NODE3:PARAM4",
+            ],
+        ),
+        (
+            "PARAM_NODE1:PARAM1",
+            ["PARAM_NODE1:PARAM1"],
+        ),
+        (
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE3:PARAM4",
+                "PARAM_NODE1X:X*",
+            ],
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE3:PARAM4",
+                "PARAM_NODE1X:X1",
+                "PARAM_NODE1X:X2",
+                "PARAM_NODE1X:X3",
+            ],
+        ),
+        (
+            ["PARAM_NODE3:P*", "PARAM_NODE2*:*2*"],
+            [
+                "PARAM_NODE2:PARAM2",
+                "PARAM_NODE22:P2",
+                "PARAM_NODE22:P22",
+                "PARAM_NODE2Y:Y2",
+                "PARAM_NODE3:PARAM4",
+            ],
+        ),
+        (
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE2Y:*",
+                "PARAM_NODE1X:X3",
+                "PARAM_NODE1X:X1",
+            ],
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE1X:X1",
+                "PARAM_NODE1X:X3",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE2Y:Y1",
+                "PARAM_NODE2Y:Y2",
+                "PARAM_NODE2Y:Y3",
+            ],
+        ),
+        (
+            "PARAM_FIELD*",
+            ["PARAM_FIELD1", "PARAM_FIELD2", "PARAM_FIELD3"],
+        ),
+        (
+            "PARAM_FIELD1",
+            ["PARAM_FIELD1"],
+        ),
+        (
+            "PARAM_GEN:*",
+            ["PARAM_GEN:0", "PARAM_GEN:1", "PARAM_GEN:2", "PARAM_GEN:3", "PARAM_GEN:4"],
+        ),
+    ],
+)
+def test_simple_config(param_group_add, expected):
+    data = {
+        "log_level": 2,
+        "correlations": [
+            {
+                "name": "some_name",
+                "obs_group": {"add": ["OBS1"]},
+                "param_group": {
+                    "add": param_group_add,
+                },
+            }
+        ],
+    }
+    conf = LocalisationConfig(observations=ERT_OBS, parameters=ERT_PARAM, **data)
+    assert len(conf.correlations) == 1
+    assert conf.correlations[0].obs_group.result_items == ["OBS1"]
```

Comment:
Is this relevant to assert? There is a separate test for observations?

Reply:
It is not relevant here. It is duplicated in test for observations.

Reply:
I have removed unnecessary assert